mobile-app/app/screens/AppNavigator/screens/Balances/screens/AddOrEditAddressBookScreen.tsx

```diff
@@ -40,7 +40,7 @@ export function AddOrEditAddressBookScreen ({ route, navigation }: Props): JSX.E
       return false
     }
     if (input.trim() === '') {
-      setLabelInputErrorMessage('Please enter an address label')
+      setLabelInputErrorMessage(translate('screens/AddOrEditAddressBookScreen', 'Please enter an address label'))
```

Comment:
i dont think it is needed, we are translating it when passing it as props to `<WalletTextInput/>`